doc/changelog

```diff
@@ -1,3 +1,14 @@
+3.1.0 (unreleased)
+  - Added possibility to enable walls building in the single player games. Useful for debug.
+  - Added ingame "debug menu". Among other things it allows to change FOV parameters on the fly. Enabled only for debug builds.
+  - Added "elevated" flag for towers. It allows them to look over opaque field when shadow caster field of view is enabled.
+  - Added enhanced fog of war preferences. Enhanced type is enabled by default.
+  - Added field of view settings. Standard radial is enabled by default and will be reloaded after every game. It allows to change FOV type for certain maps (need to enable required FOV type in the map lua-file).
+  - added "eval" cheat which allows to execute lua code from ingame console. Enabled only for debug builds.
+
+3.0.0
+  - fixme: to complete
```

Comment:
ah, yes. The changelog for 3.0.0 is in `debian/changelog`, but there it is in the debian package format

Reply:
Ok, I'll update it. 
Btw wasn't able to find war1gus changelog.

Reply:
fixed 
https://github.com/Wargus/wargus/pull/359/commits/d035882c1b7b46dab58b7e6b986a77b69a93b468